doc/classes/Node2D.xml

```diff
@@ -69,21 +71,21 @@
 			<return type="void" />
 			<param index="0" name="radians" type="float" />
 			<description>
-				Applies a rotation to the node, in radians, starting from its current rotation.
+				Rotates this node by the given angle [param radians], in radians. This operation is calculated in parent space (relative to the parent) and preserves the [member position].
```

Comment:
What is parent space?
Seems to be just local.
```C++
set_rotation(get_rotation() + p_radians);
```
That's the same as `translate()`.

Reply:
Oh god, so this is entirely different from the 3D equivalent.